pkg/apis/componentconfig/types.go

```diff
@@ -79,6 +65,11 @@ type SchedulerAlgorithmSource struct {
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta
 
+	// DebuggingConfiguration holds profiling- and debugging-related fields
+	// TODO: DebuggingConfiguration is inlined because it's been like that earlier.
+	// We might consider making it a "real" sub-struct.
+	apiserverconfig.DebuggingConfiguration `json:",inline"`
```

Comment:
these are the internal types. No need for the json tag.

Reply:
oops copy-paste error, fixed